### PR TITLE
Pattern idea: Crossing the chasm

### PR DESCRIPTION
**Pull request by MaineC**
On slack a presentation was shared on the adoption cycle for InnerSource: https://innersourcecommons.slack.com/archives/C04PXKRN4/p1636027720045600?thread_ts=1636027623.045400&cid=C04PXKRN4

This pull request is a first draft at providing a pattern for what InnerSource initiatives can do to reach teams and people at different places on the adoption curve.

ping @rrrutledge

### CODE REVIEW

File: patterns/1-initial/crossing-chasm.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+you want to establish.
+* Bring innovators together so they can support each other.
+* Write down everything that you discuss in person or in calls so others that
+lack the time to join those calls can follow along, provide their input and you
+have a document that you can share with individuals following your trail.
+
+Write down solutions to any problems you find along the way and share those
+written solutions with people following your trail.
+
+
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
```

File: patterns/1-initial/crossing-chasm.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+## Known Instances
+
+TBD
+
+## Status
+
+Initial
+
+
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md:119]:
```suggestion
```

**Comment by rrrutledge** [comment]
Neat!  Will take a look maybe next week sometime.

File: patterns/1-initial/crossing-chasm.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+## Patlet
+
+Early InnerSource experiments have been successful. Methods that were successful
+convincing early teams stop working though when scaling the initiative. This
+chasm can be crossed by using different methods to reach people at different
+stages of the innovation curve.
+
+## Problem
+
+Often InnerSource programs [start as experiments](add link here). From there
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
Often InnerSource programs [start as experiments](../2-structured/start-as-experiment.md). From there
```

File: patterns/1-initial/crossing-chasm.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+Write down solutions to any problems you find along the way and share those
+written solutions with people following your trail.
+
+### Steps for reaching early adopters/ visionaries
+
+* Share results and learnings form retrospectives of InnerSource pilots.
+* Share external material (bonus points if that is material that colleagues
+created earlier: talks, blog articles, book chapters, magazine articles, videos
+etc.)
+* Co-createa document of shared [InnerSource principles](add link here).
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
* Co-create a document of shared [InnerSource principles](explicit-innersource-principles.md).
```

File: patterns/1-initial/crossing-chasm.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+collaborative way of working.
+- Some associates refuse to invest time in experimenting with new ways of
+  working, preferring a stable environment.
+- There is too much time to get everyone on board individually.
+- Depending on where associates are on the innovation curve they are willing to
+  invest different amounts of time to master a new way of working.
+
+## Solution
+
+The [InnerSource guidance group](add link here) should make different offers for
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
The [InnerSource guidance group](../1-initial/innersource-guidance-group.md) should make different offers for
```

@MaineC did you mean this pattern?

**Comment by spier** [comment]
@MaineC thanks for pushing this idea from the slack conversation more into the open.

The pattern would be a lot easier to understand if the reader could see the adoption curve and resulting chasms visually (such visuals are referenced in the Slack thread). 

Do you think we should try to contact the original author, to see if we would be able to use those visuals and also reference them as a known instance? Of course it would also be great if the author could review this pattern draft, to make sure that we are capturing the ideas correctly.

File: patterns/1-initial/crossing-chasm.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+* Friction between teams decreases due to a common understanding.
+
+## Known Instances
+
+TBD
+
+## Status
+
+Initial
+
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md:114]:
```suggestion
```

**Comment by MaineC** [comment]
@spier First of all thank for filling in the references to existing patterns - I pushed the draft before fixing those todos ...

I agree with you that the patterns would be easier to understand if it used the visuals we had in the slack conversation from the talk. The visuals in the talk themselves actually are older coming from a book that is still on my (way too high) pile of books to read:

https://www.goodreads.com/book/show/61329.Crossing_the_Chasm

It would be awesome to have the original author of the slides engage here. I haven't seen them active on Slack (maybe I have missed that), but I'll get in touch with them through e-mail.

Thanks for the early review.

File: patterns/1-initial/crossing-chasm.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+company structures (e.g. GitHub/ GitLab instances are under a regular plan,
+accounts are easy to create and integrated with e.g. coporate single sign on
+solutions, in-house support can provide help for these tools).
+* Provide low effort opportunities for learning (e.g. put little riddles in your
+coffee kitchen or whatever the digital alternative is for your company.)
+* Ship bite sized learning pieces to inboxes.
+* Pull in external expertise - sometimes in house expertise counts much less
+than external statements.
+
+### Steps for lagers
```
Comment (lenucksi) [inline patterns/1-initial/crossing-chasm.md]: Is that supposed to be

```suggestion
### Steps for laggards
```

?

File: patterns/1-initial/crossing-chasm.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+Write down solutions to any problems you find along the way and share those
+written solutions with people following your trail.
+
+### Steps for reaching early adopters/ visionaries
+
+* Share results and learnings form retrospectives of InnerSource pilots.
+* Share external material (bonus points if that is material that colleagues
+created earlier: talks, blog articles, book chapters, magazine articles, videos
+etc.)
+* Co-create a document of shared [InnerSource principles](explicit-innersource-principles.md).
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
* Co-create a document of shared [InnerSource Principles](../1-initial/explicit-innersource-principles.md).
```

File: patterns/1-initial/crossing-chasm.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+* Co-create a document of shared [InnerSource Principles](../1-initial/explicit-innersource-principles.md).
+
+### Steps for pragmatists
+
+* Offer in-house and online training.
+* Establish local experts on topics in the teams you want to reach.
+* Provide an InnerSource strategy that is backed by higher level management.
+* Provide InnerSource cheat sheet in particular for those new to the company.
+* Provide teams with a self check to scale the initiative (see [InnerSource
+maturity model](add link here).
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
* Provide teams with a self check to scale the initiative (see [Maturity Model](../2-structured/maturity-model.md).
```

File: patterns/1-initial/crossing-chasm.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+* Overall InnerSource adoption improves.
+* Friction between teams decreases due to a common understanding.
+
+## Known Instances
+
+TBD
+
+## Status
+
+Initial
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md:113]:
```suggestion
Initial

## References

- Book: [Crossing the Chasm](https://www.goodreads.com/book/show/61329.Crossing_the_Chasm) - Geoffrey A. Moore, Regis McKenna
- [Making InnerSource & Developer Experience Real at one of Canada's Top 5 Banks](https://youtu.be/QbJ4VFk8DwA?t=549) - GitHub Universe 2021 - Anthony Vacca 

```

File: patterns/1-initial/crossing-chasm.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+## References
+
+- Book: [Crossing the Chasm](https://www.goodreads.com/book/show/61329.Crossing_the_Chasm) - Geoffrey A. Moore, Regis McKenna
+- [Making InnerSource & Developer Experience Real at one of Canada's Top 5 Banks](https://youtu.be/QbJ4VFk8DwA?t=549) - GitHub Universe 2021 - Anthony Vacca
```
Comment (spier) [inline patterns/1-initial/crossing-chasm.md]:
```suggestion
- [Making InnerSource & Developer Experience Real at one of Canada's Top 5 Banks](https://youtu.be/QbJ4VFk8DwA?t=549) - GitHub Universe 2021 - Anthony Vacca
```
```suggestion
- [Making InnerSource & Developer Experience Real at one of Canada's Top 5 Banks](https://youtu.be/QbJ4VFk8DwA?t=549) - GitHub Universe 2021 - Anthony Vacca 
```

**Comment by spier** [comment]
@MaineC are you open to merging this pattern relatively quickly, so that it becomes easier for readers to discover this pattern idea?

We can and should still improve the pattern in subsequent PRs. 
And of course if we could get other people involved that would be awesome too.

**Comment by MaineC** [comment]
I think given that all conversations are resolved we should merge this pattern to make it visible to others. We can improve it going forward from there.

**Comment by spier** [comment]
Sounds good. Done!

FYI I will also add a link to the pattern from the main [README](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/README.md), so that people can more easily find it.

**Pull request merged**